docs/quickstart/send-slack-alerts.mdx

```diff
@@ -154,8 +153,8 @@ models:
 
 <Accordion title="Custom Slack channel">
 By default Elementary uses the Slack channel that was configured in the Slack integration.
-Elementary supports configuring custom slack channels that are configured on your models / sources and snapshots.
-If you configure a custom slack channel for a model, all the test alerts that belong to this model will be sent to this custom slack channel as well.
+Elementary supports configuring custom slack channels that are configured on your models / sources / tests and snapshots.
+If you configure a custom slack channel for a model, all the test alerts that belong to this model will be sent to this custom slack channel.
 
```

Comment:
Also add - if you configure both a channel for a model and a channel for a test - the channel for a test will be used